src/Plugin_Command.php

```diff
@@ -645,6 +647,17 @@ protected function get_item_list() {
 				'title' => $details['Name'],
 				'description' => wordwrap( $details['Description'] ),
 			);
+
+			if ( null === $update_info ) {
+
+				// Get all plugin info that don't have an update.
+				$plugin_update_info = ( isset( $all_update_info->no_update[$file] ) ) ? $all_update_info->no_update[$file] : null;
+
+				// Compare version and update information in plugin list.
+				if ( null !== $plugin_update_info && version_compare( $details['Version'], $plugin_update_info->new_version, '>' ) ) {
+					$items[ $file ]['update'] = 'This version is higher than expected!';
```

Comment:
I think we should tone this down a bit so that it better fits in with the rest of the output:
```suggestion
					$items[ $file ]['update'] = 'version higher than expected';
```